packages/amplify-category-predictions/src/prompter-adapter/index.ts

```diff
@@ -0,0 +1,33 @@
+import { prompter } from 'amplify-prompts';
+
+/**
+ * This class is a wrapper around the amplify-prompts package that allows to pass a
+ * question object and decide the proper way to prompt the user based on the question type.
+ *
+ * To be integrated in AmplifyPrompter if is to be used in other packages
+ */
+class PrompterAdapter {
+  async prompt(input: any) {
```

Comment:
If possible, it would be nice to remove this wrapper entirely and convert prompts in line. If that's not possible / too much scope, provide a type for this input